src/pages/home/report/ReportActionCompose/SuggestionEmoji.tsx

```diff
@@ -232,39 +225,22 @@ function SuggestionEmoji({
 
     return (
         <EmojiSuggestions
```

Comment:
Why did we remove a lot of props here?

Reply:
Because they were never used in the component